frameworks/kafka/tests/test_overlay.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+
+import sdk_install as install
+import sdk_tasks as tasks
+import sdk_plan as plan
+import sdk_networks as networks
+import sdk_utils as utils
+
+
+from tests.test_utils import  *
+
+
+def setup_module(module):
+    install.uninstall(SERVICE_NAME, PACKAGE_NAME)
+    utils.gc_frameworks()
+
+    install.install(
+        PACKAGE_NAME,
+        DEFAULT_BROKER_COUNT,
+        service_name=SERVICE_NAME,
+        additional_options = {'service':{'virtual_network':True}})
+    plan.wait_for_completed_deployment(PACKAGE_NAME)
+
+
+def teardown_module(module):
+    install.uninstall(SERVICE_NAME, PACKAGE_NAME)
+
+
+@pytest.mark.overlay
+@pytest.mark.smoke
+@pytest.mark.sanity
+def test_service_overlay_health():
```

Comment:
I can't tell if these test are new, but if they are, please add a docstring saying what they attest to.